docs/DEVELOPMENT_GUIDE.md

```diff
@@ -0,0 +1,111 @@
+# Development Guide
+
+## How to Contribute
+
+We'd love to accept your patches and contributions to this project. There are
+just a few small guidelines you need to follow.
+
+### Submit Contributor License Agreement
+If you are not a member of the Google Genomics Organization, you will need to
+sign a Contributor License Agreement before we can review your pull request.
+Please visit
+[https://cla.developers.google.com/](https://cla.developers.google.com/clas)
+to sign.
+
+### Code reviews
+
+All submissions, including submissions by project members, require review. We
+use GitHub pull requests for this purpose. Consult
+[GitHub Help](https://help.github.com/articles/about-pull-requests/) for more
+information on using pull requests.
+
+
+## Setup
+
+### Fork the repository on Github
+Visit the [gcp-variant-transforms repository](https://github.com/googlegenomics/gcp-variant-transforms)
+to create your own fork of the repository.
+
+### Setup dev environment
+
+#### Clone the forked repository
+```
+$ git clone git@github.com:<username>/gcp-variant-transforms.git
+$ cd gcp_variant_transforms
+```
+You will also googlegenomics repository as a remote so that you can easily pull
+changes later.
+```
+$ git remote add git@github.com:googlegenomics/gcp-variant-transforms.git
+```
+
+#### Setup virtualenv
+```
+$ sudo apt-get install virtualenv
```

Comment:
virtualenv should be installed from a pip package:
 sudo apt-get install python-pip python-dev build-essential
 sudo pip install --upgrade pip
 sudo pip install --upgrade virtualenv

Reply:
Done.